app/models/course/registration.rb

```diff
@@ -0,0 +1,61 @@
+class Course::Registration < ActiveRecord::Base
+  stampable
+
+  scope :staff, -> { where(role: [roles[:teaching_assistant], roles[:manager], roles[:owner]]) }
```

Comment:
You mean adding a method that returns an array of role numbers with given role names ? What about `roles.values_at(:teaching_assistant, :manager, :owner)?

I think the roles are just enough. The name of manager may be changed to teacher or lecturer. 



> On 26 Mar 2015, at 8:27 am, Joel Low <notifications@github.com> wrote:
> 
> In app/models/course/registration.rb:
> 
> > @@ -0,0 +1,61 @@
> > +class Course::Registration < ActiveRecord::Base
> > +  stampable
> > +
> > +  scope :staff, -> { where(role: [roles[:teaching_assistant], roles[:manager], roles[:owner]]) }
> you might want to make the Roles type in CourseUser return you an array of roles that match the given characteristics.
> 
> Also, maybe we might want to reason out the categories of users. What do you propose?
> 
> \
> Reply to this email directly or view it on GitHub.
>